game.js

```diff
@@ -1,6 +1,6 @@
 function Game(player1, player2) {
-	this.player1 = Player1;
-	this.player2 = Player2;
+	this.player1 = player1;
+	this.player2 = player2;
```

Comment:
In this case, you just want to grab hold of the two objects that are passed in as arguments. Previously, you were setting this.player1 to be the constructor function instead of an "instance" of a (Human/Computer)Player.